comp/core/gui/guiimpl/views/private/js/javascript.js

```diff
@@ -63,10 +63,10 @@ function setError(status, message) {
     message = "Unable to contact the Datadog Agent. Please ensure it is running."
   }
   else if (status == 401) {
-    message = `Not logged in. Please ensure that your GUI session has not expired. (Agent replied with: ${message.trim()})`
+    message = "Not logged in. Please ensure that your GUI session has not expired. (Agent replied with: " + DOMPurify.sanitize(message.trim()) + ")"
```

Comment:
Backtick are not supported by Internet Explorer, which was preventing the load of the javascript file.